tensorflow/contrib/tensor_forest/kernels/v4/fertile-stats-resource.cc

```diff
@@ -19,6 +19,30 @@
 namespace tensorflow {
 namespace tensorforest {
 
+void FertileStatsResource::AddExampleToStatsAndInitialize(
+    const std::unique_ptr<TensorDataSet>& input_data, const InputTarget* target,
```

Comment:
why do u keep the old one (line 46) around? Where is that used
if u need the impl without random features too if it is used somewhere, u can pass a ptr to the std::vector and use it only when it is not null